scene/gui/label.cpp

```diff
@@ -447,12 +447,19 @@ void Label::regenerate_word_cache() {
 
 			if (current == '\n') {
 				insert_newline = true;
+				line_width = 0;
```

Comment:
I think you don't have to set `line_width` to 0.

`current_word_size` is set to 0 when `\n` is encountered:

https://github.com/godotengine/godot/blob/e1eb3c2fb1bcfb70204cb1d431720e316f544da1/scene/gui/label.cpp#L428

And `line_width` will be set to `current_word_size` when the `WordCache` for this newline is created:

https://github.com/godotengine/godot/blob/e1eb3c2fb1bcfb70204cb1d431720e316f544da1/scene/gui/label.cpp#L510

Reply:
@timothyqiu 
Alright, i've removed that statement, and the result of the fix did not change. :)